contracts/contracts/token/WOETH.sol

```diff
@@ -80,9 +80,11 @@ contract WOETH is ERC4626, Governable, Initializable {
         external
         onlyGovernor
     {
-        //@dev TODO: we could implement a feature where if anyone sends OETH direclty to
-        // the contract, that we can let the governor transfer the excess of the token.
-        require(asset_ != address(asset()), "Cannot collect OETH");
+        if (asset_ == address(asset())) {
+            uint256 surplus = OETH(asset()).balanceOf(address(this)) - totalAssets();
+            require(amount_ <= surplus, "Cannot collect OETH more than surplus");
```

Comment:
This is really nitpicky but Solidity is much more effective in handling strings when [they are 31 characters long or less ](https://ethereum.stackexchange.com/questions/107282/storage-and-memory-layout-of-strings). It uses 1 slot instead of 3. 

If you don't mind can you rephrase the error message to a 31 or less char long one?

Reply:
Oh yes sure, nice catch!